discoveryengine/search_sample.py

```diff
@@ -52,16 +52,44 @@ def search_sample(
         serving_config="default_config",
     )
 
-    # Refer to the SearchRequest reference for all supported fields:
+    # Optional: Configuration options for search
+    # Refer to the `ContentSearchSpec` reference for all supported fields:
+    # https://cloud.google.com/python/docs/reference/discoveryengine/latest/google.cloud.discoveryengine_v1.types.SearchRequest.ContentSearchSpec
+    content_search_spec = discoveryengine.SearchRequest.ContentSearchSpec(
+        # For information about snippets, refer to:
+        # https://cloud.google.com/generative-ai-app-builder/docs/snippets
+        snippet_spec=discoveryengine.SearchRequest.ContentSearchSpec.SnippetSpec(
+            return_snippet=True
+        ),
+        # For information about search summaries, refer to:
+        # https://cloud.google.com/generative-ai-app-builder/docs/get-search-summaries
+        summary_spec=discoveryengine.SearchRequest.ContentSearchSpec.SummarySpec(
+            summary_result_count=5,
+            include_citations=True,
+            ignore_adversarial_query=True,
+            ignore_non_summary_seeking_query=True,
+        ),
+    )
+
+    # Refer to the `SearchRequest` reference for all supported fields:
     # https://cloud.google.com/python/docs/reference/discoveryengine/latest/google.cloud.discoveryengine_v1.types.SearchRequest
     request = discoveryengine.SearchRequest(
-        serving_config=serving_config, query=search_query, page_size=10
+        serving_config=serving_config,
+        query=search_query,
+        page_size=10,
+        content_search_spec=content_search_spec,
+        query_expansion_spec=discoveryengine.SearchRequest.QueryExpansionSpec(
+            condition=discoveryengine.SearchRequest.QueryExpansionSpec.Condition.AUTO,
+        ),
+        spell_correction_spec=discoveryengine.SearchRequest.SpellCorrectionSpec(
+            mode=discoveryengine.SearchRequest.SpellCorrectionSpec.Mode.AUTO
+        ),
     )
+
     response = client.search(request)
-    for result in response.results:
-        print(result)
+    print(response)
 
-    return response.results
+    return response
```

Comment:
suggestion: please consider updating [the tests](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/00be56fecb3a8405b735a4e40910dfad115aeaca/discoveryengine/search_sample_test.py#L24), because it seems they only check if there is a response and even return type change didn't trigger failure.

Reply:
Done.